anvill/python/install.cmake

```diff
@@ -8,14 +8,20 @@
 
 find_package(Python3 COMPONENTS Interpreter REQUIRED)
 
+set(pure_lib_dest "${CMAKE_INSTALL_PREFIX}/lib/python3/dist-packages")
+
+# Only redirect installation if DESTDIR is specified, otherwise just do a normal "setup.py install"
 if(DEFINED ENV{DESTDIR})
-  set(root_parameter "--root=$ENV{DESTDIR}")
+  set(package_parameters 
+    "--single-version-externally-managed"
+    "--prefix=${CMAKE_INSTALL_PREFIX}"
+    "--root=$ENV{DESTDIR}"
+    "--install-purelib=${pure_lib_dest}"
+    )
```

Comment:
@alessandrogario can you sanity check this for me? I am afraid I may be doing the wrong thing. What I am after is a normal "setup.py install" when DESTDIR does not need to redirect installation.